cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -49,13 +50,19 @@ class CallSubqueryTest extends DocumentingTest {
         p("""The `CALL` clause is also used for calling procedures.
             #For descriptions of the `CALL` clause in this context, refer to <<query-call>>.""".stripMargin('#'))
       }
-      p("""A subquery is evaluated for each incoming input row and may produce an arbitrary number of output rows.
-          #Every output row is then combined with the input row to build the result of the subquery.
-          #That means that a subquery will influence the number of rows.
+
+      p(
+        """Subqueries which end in a `RETURN` statement are called _returning subqueries_ while subqueries without such a return statement are called _unit
+          |subqueries_.""".stripMargin)
+      p("""A subquery is evaluated for each incoming input row. Every output row of a *returning subquery* is combined with the input row to build the result of
+          #the subquery.
+          #That means that a returning subquery will influence the number of rows.
           #If the subquery does not return any rows, there will be no rows available after the subquery.""".stripMargin('#'))
+      p(
+        """*Unit subqueries* on the other hand are thought to be called for their side-effects and not for their results and therefore do not interfere with
```

Comment:
I think we can simplify this to just
```suggestion
        """*Unit subqueries* on the other hand are called for their side-effects and not for their results, and do therefore not influence
```